plugin/pkg/admission/securitycontext/scdeny/admission_test.go

```diff
@@ -139,8 +139,8 @@ func TestPodSecurityContextAdmission(t *testing.T) {
 			errorExpected: true,
 		},
 	}
-	for _, test := range tests {
-		pod.Spec.SecurityContext = &test.securityContext
+	for i, test := range tests {
+		pod.Spec.SecurityContext = &tests[i].securityContext
```

Comment:
i am missing why this change is needed?

the assignment and validation happens in each iteration of the loop.

Reply:
@derekwaynecarr, my bad. Here's the sample illustration. https://play.golang.org/p/wh5_P3NpZXl . Update the PR description as well.